README.md

```diff
@@ -365,6 +365,57 @@ it.jestPlaywrightSkip(
 )
 ```
 
```

Comment:
it's better to put after this: https://github.com/playwright-community/jest-playwright#usage-with-custom-runner